src/scripts/spells.ts

```diff
@@ -6,9 +6,15 @@ namespace Scripts {
          *
          * @param spell nazev kouzla
          * @param target na koho ma kouzlit
+         * @param forceCastInvalid novy target system v pripadne nevalidniho targetu vzdy vyhodi tercik, pro zachovani puvodniho chovani kouzleni i na waitTarget ktery neexistuje slouzi tento prepinac
          */
-        static cast(spell: string, target?: TargetEnum | string) {
-            Scripts.Utils.waitTarget(target);
+        static cast(spell: string, target?: TargetEnum | string, forceCastInvalid = true) {
```

Comment:
Asi bych tam nedaval ten treti parametr.., mozna v nejake dalsi branch odparal z utils ten waitTarget. Funkcne to bude asi stejne
```
TargetingEx.getTarget(target).isValid() && targetResult.waitTarget();
Orion.Cast(spell);

Reply:
No, ten problem je ze funkcne je to jine. Do castu sem dodelal ten muj taregting proto ze s nasazenim aliasu hover, po nem asi bude poptavka a nechtel sem aby lidi museli prepisovat hotku na muj _castSpell. 
Doded to funguje u _cast tak, ze pokud tam maji "lastattack" napriklad a ten neexistuje, nebo se dostal z range, tak to stejne dela waitTarget a zacasti to s hlaskou "You can't see the target". 
V s moji verzi, pokud ten target existuje, waittarget neprovadi = vyhodi tercik. 

koukam, ale ze spravne by tam stejne melo byt  Scripts.Utils.waitTarget(target); aby tam byl ten "legacy" support a chovalo se to stejne. Pripadne tam udelat hlasku, pokud targetResult.Success() === false.
Otazka je jak ocekvate ze se to pro vas bude chovat?